src/Datadog.Trace/Span.cs

```diff
@@ -24,6 +25,12 @@ internal Span(SpanContext context, DateTimeOffset? start)
             Context = context;
             ServiceName = context.ServiceName;
             StartTime = start ?? Context.TraceContext.UtcNow;
+
+            Log.Debug(
```

Comment:
I was actually about to comment here that I wanted to add logging about the traceId and spanId. Regardless of the user's LOGS_INJECTION_ENABLED setting, I was thinking we should always be adding the traceId and spanId with our own managed logging. Do you think we should use the event subscriber approach or do you think this beginning/end logging is sufficient?

Reply:
I think beginning and end logging is sufficient. I don't want to add TOO much to the context, as that'll be logged with everything. Maybe if we start with this and decide it's not enough we can rethink it?

Reply:
Sounds good to me, this is already going to be great information to have!